src/target/adiv5.c

```diff
@@ -586,10 +584,10 @@ static void adiv5_component_probe(ADIv5_AP_t *ap, uint32_t addr, const size_t re
 				arm_component_lut[i].arch_id != arch_id)
 				continue;
 
-			DEBUG_INFO("%s%d 0x%" PRIx32 ": %s - %s %s (PIDR = 0x%02" PRIx32 "%08" PRIx32 "  DEVTYPE = 0x%02" PRIx8
+			DEBUG_INFO("%s%d 0x%" PRIx32 ": %s - %s %s (PIDR = 0x%016" PRIx64 "  DEVTYPE = 0x%02" PRIx8
```

Comment:
Doesn't newlib not support > 32 bit variables on printf? Or is this hosted only code (on mobile, hard to see context)

Reply:
Good question.. we'll revert this particular change if it doesn't, we'll test a debug build of this PR in the morning to check

Reply:
This bothered us enough that we actually already checked and you're correct, newlib doesn't support 64-bit printf 😞 and people wonder why we roll our own when we have a choice